tests/test_data/lecker_1.testhtml

```diff
@@ -276,20 +276,20 @@ jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","setHasJsCookie"
            bdu_screen_init();
            
            if (bdu_screen == 'smartphone') {
-                var iam_data = { 
-"st":"moblecke",
-"cp":"LE_Rezepte_Einzel", 
-"sv":"mo",
-"sc":"yes"
-} 
+                var iam_data = { 
+"st":"moblecke",
+"cp":"LE_Rezepte_Einzel", 
+"sv":"mo",
+"sc":"yes"
+} 
 iom.h(iam_data);
             } else {
-                var iam_data = { 
-"st":"leckerde",
-"cp":"LE_Rezepte_Einzel", 
-"sv":"in",
-"sc":"yes"
-} 

```

Comment:
Odd - it looks like something might have rewritten a few line endings in this test file.  Not hugely important, but would be nice to remove from the changeset if possible.

Reply:
i ran black and flake a bunch of times to get the formatting the way its supposed to but did not look at the test file again so thats probably what caused it, will fix it if pre commit does not kill it automatically again

Reply:
updated using the latest data from lecker.de, updated the rating as well since it changed since then

Reply:
:shrug: `black` and `flake8` shouldn't modify HTML content, so it may have been an editor setting